src/Sentry.Unity/SentryUnity.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using Sentry.Extensibility;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Sentry.Unity
+{
+    public sealed class SentryUnity : IDisposable
+    {
+        private static SentryUnity? Instance;
+
+        internal readonly ErrorTimeDebounce ErrorTimeDebounce = new(TimeSpan.FromSeconds(1));
+        internal readonly LogTimeDebounce LogTimeDebounce = new(TimeSpan.FromSeconds(1));
+        internal readonly WarningTimeDebounce WarningTimeDebounce = new(TimeSpan.FromSeconds(1));
+
+        internal UnitySentryOptions? Options { get; private set; }
+
+        internal IEventCapture EventCapture { get; set; } = new EventCapture();
+
+        public static SentryUnity Init(UnitySentryOptions unitySentryOptions, Action<SentryUnity>? unityConfigure = null)
+        {
+            // Force dispose if new Init
+            if (Instance != null)

```

Comment:
This is racy. If `Init` is called from different threads this wouldn't work well.

Check Sentry .NET:

https://github.com/getsentry/sentry-dotnet/blob/main/src/Sentry/SentrySdk.cs#L102-L104